### ISSUE TITLE
PII Exposure

### ISSUE THREAD

**Issue by JulianGindi** [sgaudaen/VirtualTipJar#2]
There is a ton of personal data stored publicly in this repo. This data should typically be kept out of the repo.

https://github.com/sgaudaen/VirtualTipJar/blob/master/Assets/data.csv

**Comment by jakerella** [comment]
I agree... they are using a public google doc as the data source: https://docs.google.com/spreadsheets/d/1tz2uyhgy3MsBS68MHPzO8H455_879fqfIaRPyUUw3QE/edit#gid=0

**Comment by sgaudaen** [comment]
If the data wasn't already public, I would have done more to hide it. I left it in the repo so if someone wanted to deploy their own they already had data in place.
